modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -18,4 +18,82 @@ Depending on your {ibm-z-name} environment, you can choose from the following op
 [NOTE]
 ====
 Currently, ISO boot support on {ibm-z-name} (`s390x``) is available only for {op-system-base-full} KVM, which provides the flexibility to choose either PXE or ISO-based installation. For installations with z/VM and Logical Partition (LPAR), only PXE boot is supported.
+====
+
+[id="networking-reqs-ibmz_{context}"]
+==  Networking requirements for {ibm-z-title}
+
+In {ibm-z} environments, advanced networking technologies like Original Storage Architecture (OSA), HiperSockets, and Remote Direct Memory Access (RDMA) over Converged Ethernet (RoCE) require specific configurations that deviate from the standard settings used in the Agent Based Installer deployments. These overrides are necessary to accommodate their unique requirements and ensure a successful and efficient deployment on {ibm-z}.
+
+The following table lists the network devices that are supported for the network configuration override functionality:
+
+[cols="3,2,2,2,2", options="header"]
+|====
+| Network device 
+| z/VM  
+| KVM 
+| LPAR Classic 
+| LPAR Dynamic Partition Manager (DPM)
+
+| Original Storage Architecture (OSA) virtual switch 
+| Not needed
+| --
+| Not needed
+| Not needed
+
+| Direct attached OSA 
+| Supported
+| Only through a Linux bridge 
+| Supported
+| Not required
+
+| RDMA over Converged Ethernet (RoCE) 
+| Not needed
+| Only through a Linux bridge 
+| Not needed
+| Not needed
+
+| HiperSockets 
+| Supported 
+| Only through a Linux bridge 
+| Supported 
+| Not needed 
+|====
+
+[id="configuring-network-overrides-ibmz_{context}"]
+=== Configuring network overrides in {ibm-z-title}
+
+You can specify a static IP address on {ibm-z} machines that uses Logical Partition (LPAR) and z/VM. This is specially useful when the network devices do not have a static MAC address assigned to them. 
+
+If you have an existing `.parm` file, edit it to include the following entry:
+
+[source,terminal]
+----
+ai.ip_cfg_override=1
+----
```

Comment:
If this overall section is moved to a procedure module, this should be made into a single step procedure:
```suggestion
.Procedure

* If you have an existing `.parm` file, edit it to include the following entry:
+
[source,terminal]
----
ai.ip_cfg_override=1
----
```